### PR TITLE
Better: Structured contents item payload fields can only contain ASCII chars RD-20221

### PR DESCRIPTION
**Pull request by kalilz4485**

### CODE REVIEW

File: docs/interactions/structured-messages/select.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content.items`** | Array | An array of items representing the options presented to the customer. A maximum of 13 items is supported. |
 | **Item Settings** | | |
 | **`structured_content.items.title`** | String | The title of the item. Limited to 80 characters. |
-| **`structured_content.items.payload`** | String | **Optional**. Payload that will be returned with the structured response. Limited to 1000 characters. |
+| **`structured_content.items.payload`** | String | **Optional**. Payload that will be returned with the structured response. Limited to 1000 characters. Can only contain **ASCII** characters. |
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/select.md:48]: Screenshot from 2022-03-17 10-19-29

File: docs/interactions/structured-messages/list.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content.items`** | Array | An array of items representing the options presented to the customer. A maximum of 10 items is supported. |
 | **Item Settings** | | |
 | **`structured_content.items.title`** | String | The title of the item. Limited to 80 characters. |
-| **`structured_content.items.payload`** | String | **Optional**. Payload that will be returned with the structured response. Limited to 200 characters.<br>Automatically gets populated as a random hex if blank. |
+| **`structured_content.items.payload`** | String | **Optional**. Payload that will be returned with the structured response. Limited to 200 characters.<br>Automatically gets populated as a random hex if blank. Can only contain **ASCII** characters. |
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/list.md:53]: Screenshot from 2022-03-17 10-19-15

File: docs/interactions/structured-messages/template.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **Item Settings** | | |
 | **`structured_content.items.type`** | String | The type of the button. Can be **url** or **reply**. |
 | **`structured_content.items.title`** | String | The title of the item. Limited to 80 characters. |
-| **`structured_content.items.payload`** | String | **Optional** when the type is "reply". **Ignored** when the type is "url". Payload that will be returned with the structured response. Limited to 1000 characters. |
+| **`structured_content.items.payload`** | String | **Optional** when the type is "reply". **Ignored** when the type is "url". Payload that will be returned with the structured response. Limited to 1000 characters. Can only contain **ASCII** characters. |
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/template.md:53]: Screenshot from 2022-03-17 10-18-58

**Comment by onima** [comment]
LGTM 👍

**Pull request merged**